world/quests/test/task-demo.lisp

```diff
@@ -0,0 +1,185 @@
+| ! eval (activate 'fi-fire)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-fire :interactable fi :dialogue "
+~ player
+| I found the remnants of another human camp. Recently vacated.
+~ fi
+| I think it's fair to say that Harris hasn't been completely honest with us.
+~ player
+| You think it's a trap?
+~ fi
+| There's one way to find out.
+~ player
+| Yes... Lucky me.
+~ fi
+| You can handle yourself.
+| [has-more-dialogue Next? | Sayonara.]")
+(quest:interaction :name beacon-check :interactable beacon :dialogue "
+~ player
+| (How did this get all the way up here?)
+| (... Look at that view... Somebody's thinking place?)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name corpse-check :interactable corpse :dialogue "
+~ player
+| (Remains: human. Dismembered, and already picked clean.)
+| (An animal attack? Or those rogue robots, perhaps.)
+| ! eval (complete 'fi-check)
+")
+; \/ query if already know of landslide var, and tweak text e.g. "This is the other side of the landslide"
+(quest:interaction :name cave-in-check :interactable cave-in :dialogue "
+~ player
+| (The cave collapsed here. This is where the humans came through - I can see more bodies underneath.)
+| (The scratches in the rocks suggest combat occurred - swords, gunfire, and finger nails...)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name android-check :interactable android :dialogue "
+~ player
+| (It's the android... It's been destroyed. Not even Catherine could fix this kind of damage.)
+| Hello, sestra... What happened to you?
+| (More human remains. It appears you didn't go down without a fight.)
+| (The memory replay seems to be intact.)
+| Excuse me, sestra... I just need to retrieve this... --- Thank you.
+| (I'd better get this back to Fi.)
+| ! eval (complete 'fi-check)
+| ! eval (activate 'fi-outcome)")
+(quest:interaction :name home-check :interactable home :dialogue "
+~ player
+| (They made a home here. Who was Harris to take that away?)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name home-tent-check :interactable home-tent :dialogue "
+~ player
+| (Androids do use bedrolls. At least, this one did.)
+| ! eval (complete 'fi-check)
+")
+(quest:interaction :name fi-outcome :interactable fi :dialogue "
+~ fi
+| Is everything okay?
+~ player
+| I found the android.
+~ fi
+| Oh... -- I'm so sorry...
+| What happened?
+~ player
+| They're offline and irreparable.
+| By the lay of the carnage in there, I'd say Harris and company ambushed them.
+| They fought, and the cave destabilised and collapsed. The humans were killed in the cave in - all but one.
+| The android then killed Harris - not just killed him, but dismembered him.
+| They spent their final seconds crawling back into their home, where they offlined.
+~ fi
+| -- You think Harris tricked us into coming here? To seize you too?
+~ player
+| Perhaps.
+~ fi
+| I see you have the memory replay.
+~ player
+| *Holds the small metal chip aloft*
```

Comment:
`Got it right here.`

Reply:
I do like the freedom the action descriptions would give us to show little visual actions from time to time, without having to rely on speech to convey it instead (and without having to animate it necessarily - though bonus if the art budget can do that :) ). Though is perhaps a wider discussion, including on how a narrative delivery method like that should be implemented in a system so far designed for speech

Reply:
I think the Ace Attorney games are a perfect example on how to handle this situation. They have very, very few animations for every character, and I don't believe they ever rely on asterisks to express actions, instead using UI elements, or context clues in a very effective way.

Here's a good video on their animations in general: https://www.youtube.com/watch?v=zV8sjRTiWC8